assets/helpers/payPalContributionsCheckout/payPalContributionsCheckoutActions.js

```diff
@@ -1,6 +1,7 @@
 // @flow
```

Comment:
Why are we leaving this file? If we are going to get rid of the state of PayPalContributionsCheckout we should delete this file, as well as the reducer and its tests.

Reply:
I think because it may be wired up elsewhere.